README.md

```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-# Math Magicians: setup
+# Getting Started with Create React App
 
-This project is about creating a math calculator with 3 pages but this time builded with react.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Built With
+## Available Scripts
 
```

Comment:
- [ ] Please make sure that the readme file is like the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md).

Reply:
CHanges are done, thank you